language/stdlib/modules/LibraAccount.move

```diff
@@ -970,7 +887,8 @@ module LibraAccount {
         // Verify that the transaction sender's account exists
         assert(exists_at(transaction_sender), EPROLOGUE_ACCOUNT_DNE);
 
-        assert(!account_is_frozen(transaction_sender), EPROLOGUE_ACCOUNT_FROZEN);
+        assert(!AccountFreezing::account_is_frozen(transaction_sender), EPROLOGUE_ACCOUNT_FROZEN);
+        assert(!VASP::is_frozen(transaction_sender), EPROLOGUE_ACCOUNT_FROZEN);
```

Comment:
Might be worth adding a comment explaining why there's a separate check for VASPs.